script/Ciel/Day 8/08_01_CIEL08_1E.txt

```diff
@@ -237,63 +284,71 @@ Isn't that right, Ciel-san?"
 [sha:318eed43a12863569f3e2c0de21f81036001e2ae]{
 -- Page 150, Offset 18652.
 -- 「そうですね。たとえ自業自得だとしても、私たちの前で死なれるのは困ります。
-"Yes. Though it would not be undeserved, we would be put in a difficult situation if he were to die in front of us.
+"Indeed.
+ It'd be a problem if he died on our watch, even if he did deserve it.
 }
 [sha:8e4137ce5b9f376d73c5b2fca747af4d4c892bc9]{
 -- Page 150, Offset 18653.
 -- 今のように、話を盗み聞かれるのも面白くありませんが」
-It is also unpleasant to have one's conversation be eavesdropped on, like you just did."
+I can't say I appreciate being eavesdropped on like that either."
 }
 [sha:7860a007dbb1ed56777a189dee7dde11da8b5d0d]{
 -- Page 151, Offset 18654.
 -- 先輩は淡々とそんな事を言った。
-Senpai says indifferently.
+Senpai's voice is dispassionate.
 }
 [sha:e503520bdab15b7075fa4a5041b250b401ae273f]{
 -- Page 151, Offset 18655.
 -- 朝の不機嫌な先輩とはまた違う、素っ気のない態度だった。
-She's cold to me, but in a way that's also different from the bad mood she had this morning.
+She's acting coldly, but this is different to the bad mood she'd been in this morning.
 }
 [sha:3848c42ec3a9708b1b2ddc70c6f7e016fc923b06]{
 -- Page 151, Offset 18656.
 -- ……むむ。どう見ても演技だと分かるけど、盗み聞き扱いはちょっと酷いと思う。
-// Is 演技 referring to Ciel? I'm not 100% sure.
-...Mhmm, I get that this is an act, but it's a bit harsh to treat that as eavesdropping.
+// G: Is 演技 referring to Ciel? I'm not 100% sure.
+// R: Pretty sure
+Hmph... It's pretty clear this is all an act,
+ but it still strikes me as a bit harsh to consider what I did 'eavesdropping'.
 }
 [sha:3af558f27d53890b4968cc1a4682cd6cf06596e5]{
 -- Page 152, Offset 18657.
 -- 「すみません、すぐ倒れる貧弱な体質で。先輩ほど頑丈じゃないんです」
-"I'm sorry that I have a weak body that collapses so easily. I'm not as sturdy as you, Senpai."
+"I'm sorry that I'm weak and prone to collapsing.
+ We can't all be as sturdy as you are, Senpai."
 }
 [sha:2a52a02164a969fa7191464f5bb5a32a76bbbc36]{
 -- Page 153, Offset 18658.
 -- 「わ、わたしの体の話はしていませんっ。
-"W-We are not going to talk about my body.
+"D-Don't you drag my body into this.
 }
 [sha:99ef217b35a3422a375adbd57c5b670fd748506d]{
 -- Page 153, Offset 18659.
 -- ……とにかく、しばらく大人しくしているように。街はまだ危険な状態です。勝手に出歩いて、勝手に襲われる人を助けるほど、私たちも暇ではありませんから」
-...Anyway, just listen to us for now. It's still dangerous out there. I don't have the leisure to help someone who's irresponsibly strolling about and who gets irresponsibly attacked."
+Look, just behave and stay put for a little while.
+ It's still dangerous to be out and about in the city.
+ I don't have time to run around bailing out people who wander off on their own and get themselves attacked."
 }
 [sha:aeaf64ddada25042f2a7d30bdcf3ab5e7f7464f9]{
 -- Page 154, Offset 18660.
 -- 「あら。志貴クンなら死者ぐらいは返り討ちにできるんじゃない？　というか、先生ひらめいちゃった。
-"Oh? If the Dead attack Shiki-kun, they'll be the ones killed, no? And I just had an idea!
+"Huh? If any of the Dead attack Shiki-kun then that's their funeral, no?
```

Comment:
Maybe use interjection 'why' here? Both 'oh' and 'huh' strikes me as a little out of place in this sentence.

Reply:
agree about "huh" but "ooh" works for me

Reply:
Not sure I love `why` because the preceding sentence is "We don't have time to run around bailing [people out]", and so the `why` associates more strongly to that than it does the `It's still dangerous to be out and about in the city.` Maybe `Is it?` if we really don't like `huh`